src/main/java/blackjack/controller/BlackJackController.java

```diff
@@ -32,13 +33,30 @@ private <T> T repeat(Supplier<T> supplier) {
     }
 
     public void run() {
+        Map<String, Integer> bettingMoney = getBettingMoney(blackjackGame.getPlayersName());
         blackjackGame.drawInitCard();
         printInitStatus(blackjackGame.getParticipantsInit());
         drawMoreCardForPlayers(blackjackGame.getPlayersName());
         InputView.closeScanner();
         drawMoreCardForDealer();
         printAllStatus();
-        printGameResult();
+        printGameResult(blackjackGame.getParticipantsProfit(bettingMoney));
+    }
+
+    private Map<String, Integer> getBettingMoney(List<String> playersName) {
+        Map<String, Integer> bettingMoney = new HashMap<>();
```

Comment:
```suggestion
        Map<Player, Integer> bettingMoney = new HashMap<>();
```
형식으로 객체에 매핑하고, 도메인 요소로 가지고 있어도 좋을 듯 싶네요!

Reply:
`String`을 사용한 이유는 컨트롤러가 domain을 모르게 하기 위해서였습니다. 이 부분은 그대로 두고, getbettingMoney()이 dto를 리턴하도록 수정하였습니다!

Reply:
제가 의도했던 것은 이런 로직이 BlackjackGame에 들어가서 객체로 나올 수 있고, dto로 변환시에는 도메인이 아닌 view에 사용될 dto용 객체를 만드는 부분을 의도했던 것 같은데요. 혹시, BlackjackGame에 넣기 어려운 부분이 있었나요..?